WNPRC_Purchasing/resources/queries/ehr_purchasing/purchasingRequestsOverview.query.xml

```diff
@@ -35,6 +35,10 @@
                     </column>
                     <column columnName="OtherAcctAndInves"/>
                     <column columnName="requester"/>
+                    <column columnName="orderDate"/>
+                    <column columnName="reorderAction">
+                        <required>false</required>
```

Comment:
Do you need both this and the setRequired(false) in WNPRC_PurchasingCustomizer.java?